apps/server/src/modules/h5p-editor/service/h5p-editor.service.ts

```diff
@@ -1,25 +1,36 @@
 import { H5PEditor, cacheImplementations } from '@lumieducation/h5p-server';
 
+import { IH5PEditorOptions } from '@lumieducation/h5p-server/build/src/types';
 import { ContentStorage } from '../contentStorage/contentStorage';
 import { LibraryStorage } from '../libraryStorage/libraryStorage';
+import { Translator } from './h5p-translator.service';
 import { TemporaryFileStorage } from '../temporary-file-storage/temporary-file-storage';
 import { h5pConfig, h5pUrlGenerator } from './config/h5p-service-config';
 
 export const H5PEditorService = {
 	provide: H5PEditor,
 	inject: [ContentStorage, LibraryStorage, TemporaryFileStorage],
-	useFactory(contentStorage: ContentStorage, libraryStorage: LibraryStorage, temporaryStorage: TemporaryFileStorage) {
+	async useFactory(
+		contentStorage: ContentStorage,
+		libraryStorage: LibraryStorage,
+		temporaryStorage: TemporaryFileStorage
+	) {
 		const cache = new cacheImplementations.CachedKeyValueStorage('kvcache');
 
+		const h5pOptions: IH5PEditorOptions = {
+			enableHubLocalization: true,
+			enableLibraryNameLocalization: true,
+		};
+
 		const h5pEditor = new H5PEditor(
 			cache,
 			h5pConfig,
 			libraryStorage,
 			contentStorage,
 			temporaryStorage,
-			undefined,
+			await Translator.translate(),
```

Comment:
I dont like the look of an await inside of a const declaration, maybe you can just do something like `const translator = Translator.translate()` and then set that in the new H5P...

Reply:
@MartinSchuhmacher Also in this solution, an 'await' is needed inside a const declaration:
`const translator = await Translator.translate();`